Basic/DyanmicPrefabs/Assets/Scripts/01_Server authoritative dynamic prefab loading across peers/AppController.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using Random = UnityEngine.Random;
+
+namespace Game
+{
+    public sealed class AppController : NetworkBehaviour
```

Comment:
@fernando-cortez we might want to look at how we split this code base once we go the "multi-scene" route.
Might be worth it to have very separate assemblies for each use case, to make things easier to find. Right now this AppController is handling a bit too many things like UI logic, connection logic, etc. We can look at this together when you get on this.

Reply:
AppController is indeed a bit fat for a pure appcontroller, but idk - it's a toy sample. Anyhow, slice it how you think is best :)

Reply:
This will be further addressed in a follow up PR. Connection management has helped split this code up already, and this class will be trimmed down even more with the UI pass (work for future UI PR).